feat(ui): cube swiper를 추가하기 위해 cube effect을 추가하고, style을 갱신한다

sukjae opened the pull request:
## 요약

- 추가하고자 하는 effect [문서 예시](https://swiperjs.com/demos/#3D_cube_effect)
- 이를 위해 style을 추가하고, 추가적으로 [Swiper](https://www.npmjs.com/package/swiper)의 ~~버전을 올림~~ 해당 effect를 esm 으로 import 함

## 리뷰 받고자 하는 부분

- [Swiper](https://www.npmjs.com/package/swiper)를 이용한 Swiper(class101-ui의)의 사용이 많은것으로 알고 있습니다.
  - 이에 대해 어느 부분에 영향을 줄지 파악하기 어려워, 구현하셨던 분이 리뷰해주시면 감사하겠습니다. 
- ~~[Swiper](https://www.npmjs.com/package/swiper)의 버전을 올렸습니다.~~
  - ~~이를 의도하여 올리지 않았던 것이라면, 코맨트 부탁드립니다~~
- [Swiper](https://www.npmjs.com/package/swiper)와 사용될 Css파일(`swiperDefaultStyle.ts`)를 변경하였습니다. 
  - ~~그 과정에서, 기존 파일의 history를 보았고, 기존에 [Swiper](https://www.npmjs.com/package/swiper)에서 제공되던 스타일과 크게 다르지 않아 새로운 버전으로 덮어 씌었습니다.~~
  - ~~혹시 이 부분에서 제가 모르는 히스토리가 있다면, 공유 부탁드립니다.~~ 
  - Swiper에서 3D와 cube와 관련된 부분만 추가하였습니다
- 아래 의견에 따라 [Swiper](https://www.npmjs.com/package/swiper)의 모듈을 추가하였습니다
  - 이 부분 적절히 사용된것인지 확인 부탁드립니다
- lock 파일이 갱신되었습니다. 
  - 별도로 추가로 설치하지 않았으나, 로컬 node에 의해 그런것이라 생각되는데, 혹시 문제의 여지가 있는지 궁금합니다

sim-zi commented:
swiper는 번들사이즈가 매우 크기 때문에 전체 번들을 사용하면 웹앱의 초기 로딩 속도가 많이 느려집니다
그래서 사용하는 코드만 번들에 포함되도록 esm 을 사용하거나 custom build를 만들어서 사용해야하는데요.
이전에 cube effect 를 사용하지 못했던 것이 이 이유 때문입니다. cube effect와 관련된 js와 css를 포함하지 않도록 해서 번들 사이즈를 최적화한 것이었어요.

그래서 가급적 원래처럼 필요한 코드만 번들에 포함되도록 esm 으로 import 하도록 하는 게 좋겠습니다.

https://swiperjs.com/api/#custom-build

ddalpange commented on package.json:
```diff
@@ -106,7 +106,7 @@
     "polished": "^3.4.1",
     "react-popper": "^1.3.7",
-    "swiper": "4.4.6"
+    "swiper": "^5.3.8"
```
^를 쓰지 말고 항상 버전을 고정해주세요.

sukjae replied:
넵! 
이 부분 다시 되돌려서, 버전 올리지 않고 진행하려 합니다~

sukjae committed:
feat(ui): cube effect module를 포함한다

- 기존에 번사이즈 최적화를 위해서 cube effect가 포함되지 않았음
  - cube effect를 사용하기 위해서 모듈을 추가함
  - 참고: https://github.com/pedaling/class101-ui/pull/241\#issuecomment-628346794

sukjae committed:
docs(ui): cube 효과의 swiper 예시를 추가한다

sim-zi reviewed (approved):
lgtm!

다만, 이 변경 이후 class101-ui의 swiper를 사용하는 모든 애플리케이션에서 cube effect를 사용하지 않더라도 번들에 cube effect 관련 코드를 포함하게 된다는 점은 유의해두세용.

필요한 코드만 사용하게 할 수 있는지 알아보려면, `Swiper.use([...])` 를 다이나믹하게 설정할 수 있는지 찾아보면 좋을 거 같습니다.

chiabi commented:
@sukjae 요것은 계속 진행되는 걸까요??? 
충돌도 확인해주세용
